architecture-decision-records/0011-base25-encoding-for-consignment-reference.md

```diff
@@ -0,0 +1,88 @@
+# base25 encoding for consignment references
+
+**Date:** 2020-11-13
+
+## Context
+
+As part of the 'transfer summary' page, the user will see a 'consignment reference' to identify their consignment (which can be useful if a user needs to contact us about a specific consignment). We came to the decision that we liked the format:
+
+`TDR-YEAR-base25reference`
+
+`TDR` - since the consignment will be transferred using the TDR service
+
+`YEAR` - year the consignment is created (not completed)
```

Comment:
It's probably worth mentioning that the reason we use the consignment creation year is that a transfer might takes days or weeks to finish, so we don't know the completion year when we generate the reference. We have to generate the reference at the start of the process so we can display it on pages like the transfer summary page and the transfer history page.